release/python/l2tester/send_frames.py

```diff
@@ -73,14 +73,15 @@ def send_frame(frame_mapping):
 ## Send And Check frames ###########################################################################
 
 def send_and_check(send_frames, expected_frames, ignore_unexpected_frames=False, timeout=1.0,
-                   do_not_log_error_on_error=False, print_raw_packets=False):
+                   do_not_log_error_on_error=False, print_raw_packets=False, ignored_types=None):
```

Comment:
Looks like the ignored_types would be something like ['TCP', '0x8100', 'ICMPv6ND_RA']